devlab/tests/base.py

```diff
@@ -31,6 +32,36 @@
 OPENSTACK_RELEASES = {'192.168.1.2': 'grizzly',
                       '192.168.1.3': 'icehouse',
                       '192.168.1.8': 'juno'}
+SWIFT_AUTH_VERSION = '2'
+
+
+class SwiftConnection(object):
+    def __init__(self, auth_url, username, password, tenant):
+        self.auth_url = auth_url
+        self.username = username
+        self.password = password
+        self.tenant = tenant
+
+    def __enter__(self):
+        class Swift(object):
```

Comment:
second: Just think that no-one will use the class directly, you need to believe most of developers are smart
first: create a function with creating an connection and close it after